packages/expo-contacts/src/Contacts.ts

```diff
@@ -222,6 +222,10 @@ export type UrlAddress = {
  * > On Android you can get dimensions using [`Image.getSize`](https://reactnative.dev/docs/image#getsize) method.
  */
 export type Image = {
+  /**
+   * A **local image URI**.
+   * > Note: if you have a remote URI, you can download it first using [`FileSystem.downloadAsync`](https://docs.expo.dev/versions/latest/sdk/filesystem/#filesystemdownloadasyncuri-fileuri-options).
```

Comment:
Since this is in the TypeScript file, I used an external link since I didn't know if [internal links](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#linking-to-other-docs) would work in this context. Also I found other examples of this in `packages/`

Reply:
```suggestion
   * > Note: if you have a remote URI, you can download it first using [`FileSystem.downloadAsync`](/versions/latest/sdk/filesystem/#filesystemdownloadasyncuri-fileuri-options).
```

After the docs are generated (for example, in `public/static/data` the internal linking works).